VisualizationBase/src/ViewItemManager.cpp

```diff
@@ -135,6 +135,24 @@ void ViewItemManager::removeAllViewItems()
 	currentViewItem_ = nullptr;
 }
 
+
+void ViewItemManager::removeViewItem(ViewItem* view)
```

Comment:
This method has to be more symmetric to the `addViewItem` one. Make sure to call `scene_->removeTopLevelItem`.